src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -902,7 +902,7 @@ public FeedbackResponseAttributes getActualResponse(FeedbackResponseAttributes r
 
     public String getNameForEmail(String email) {
         String name = emailNameTable.get(email);
-        if (name == null) {
+        if (name == null || name.equals(Const.USER_IS_MISSING)) {
```

Comment:
If needed, it can be done in the same class as it's not used anywhere. But if the idea is to reduce duplications, the methods `getNameForEmail`, `getLastNameForEmail`, `getRecipientNameForResponse`, and `getGiverNameForResponse` (essentially all methods that has that condition check) are all ~95% duplications, not just this condition check.